app/models/application.rb

```diff
@@ -161,6 +161,14 @@ def find_all_nearest_or_recent
     end
   end
 
+  def official_submission_period_expired?
+    if on_notice_to && Date.today > on_notice_to
+      true
+    else
+      false
+    end
```

Comment:
That produces a different result in fact, because if on_notice_to isn't set it returns nil. For our use of this, we want to return false—we don't know if it's expired so we cannot say it's true.

It can be compressed to:

  on_notice_to && Date.today > on_notice_to ? true : false